gui/src/Layout.js

```diff
@@ -52,19 +50,18 @@ const Layout = useHooks(props => {
 
   const {classes, redirectTo} = props
 
-  return (<Router>
-    <Grid container>
+  return (<BrowserRouter>
```

Comment:
In this case, BrowserRouter would be rendering in the node stage as well. You need to detect which stage the code is in and use the right router and history combo.  Static router for node and browser router for non-node. You can do this by checking for the existence of the Dom. Here is how the old react static worked: 

https://github.com/nozzle/react-static/blob/v5/src/client/components/Router.js#L146

You'll also need to provide the route for the static router. The old react static did it this way: 

https://github.com/nozzle/react-static/blob/v5/src/client/components/Router.js#L180

But you'll likely need to access the staticInfo
To do that. Now that I think of it, there is no API currently to access that directly. I'm going to take a closer look at this in the morning and help you find a solution.

Reply:
@reach/router is actually much easier. It requires much less babysitting for static rendering. You'll be pleasantly surprised.

Reply:
The new react-static package works well! Thank you again. 🎉